vms/avm/environment_test.go

```diff
@@ -124,7 +120,9 @@ func setup(tb testing.TB, c *envConfig) *environment {
 	}
 
 	genesisBytes := buildGenesisTestWithArgs(tb, genesisArgs)
-	ctx := newContext(tb)
+
+	ctx := snowtest.Context(tb)
+	ctx.ChainID = ctx.XChainID
```

Comment:
It'd be better to specify the ChainID in the context constructor, rather then modify the context ex-post.
We may export "X" "P" and "C" as constants and turn context creation to something like  `snowtest.Context(tb, snowtest.XChain)`.
The reason for this is that context has a bunch of interrelated attributes. These kind of ex-post overwrites are difficult to maintain. The best is to have a single function responsible for setting the context right and have tests just read the context then